docker-compose.yml

```
@@ -221,7 +249,7 @@ services:
     container_name: aerie_hasura
     depends_on: ["postgres"]
     environment:
-      AERIE_DATABASE_URL: "postgres://${AERIE_USERNAME}:${AERIE_PASSWORD}@postgres:5432/aerie?options=-c%20search_path%3Dutil_functions%2Chasura%2Cpermissions%2Ctags%2Cmerlin%2Cscheduler%2Csequencing%2Cpublic"
+      AERIE_DATABASE_URL: "postgres://${AERIE_USERNAME}:${AERIE_PASSWORD}@postgres:5432/aerie?options=-c%20search_path%3Dutil_functions%2Chasura%2Cpermissions%2Ctags%2Cmerlin%2Cscheduler%2Csequencing%2Cpublic%2Cactions"
```

`actions` and `public` should be reversed, otherwise you're telling Hasura to let objects in the `public` schema overshadow objects in the `actions` schema when the schema isn't explicitly specified (this shouldn't _currently_ cause a bug as we keep our `public` schema empty atm, but it is not advised).

```suggestion
      AERIE_DATABASE_URL: "postgres://${AERIE_USERNAME}:${AERIE_PASSWORD}@postgres:5432/aerie?options=-c%20search_path%3Dutil_functions%2Chasura%2Cpermissions%2Ctags%2Cmerlin%2Cscheduler%2Csequencing%2Cactions%2Cpublic"
```

---

Got it, I updated the url

---

Thanks!
Follow-up to this: I just realized that only the `develop` `docker-compose.yml` has its `AERIE_DATABASE_URL` updated. Please update this in the other `docker-compose`s.

---

I believe I got them all.